src/main/java/racingcar/model/ResultView.java

```diff
@@ -0,0 +1,17 @@
+package racingcar.model;
+
+import racingcar.CarList;
+
+public class ResultView {
+    private static final String EXECUTION_RESULT = "실행 결과";
+    private static final String BAR = "-";
+
+    public static void printStartComment() {
+        System.out.println(EXECUTION_RESULT);
+    }
+
+    public static void print(CarList cars) {
+        cars.currentCars().forEach(car -> System.out.println(BAR.repeat(car.currentPosition())));
```

Comment:
```suggestion
        cars.currentCars()
            .forEach(car -> System.out.println(BAR.repeat(car.currentPosition())));
```
가독성을 위해 한줄에 하나의 점 `.`을 사용하시는 것을 추천드려요~

Reply:
넵넵 :)